### DISCUSSION TITLE
Weekly #5

### DISCUSSION THREAD

**Discussion by Ninetilt** [Ninetilt/Project-TDC#7] [General]
### **Übersicht**
Diese Woche markiert den Abschluss unseres zweiten Sprints. Wie wollen euch in diesem kleinen Update eine erste Demo unserer App auf "Hello World" Niveau zeigen und auch ein Paar Worte über den vergangenen Sprint verlieren.

### **Demo**
Die User-Story die wir mit unserer Demo umgesetzt haben kommt aus Developersicht. Für einen einfachen und nahtlosen Einstieg in die Programmieren der Funktionen der App war uns zunächst wichtig das Projekt so einzurichten, dass man die App während der Entwicklung direkt mit einem Mobilgerät synchronisieren kann und mit Hilfe dessen auch debuggen kann. In unserer Demo haben wir mit .NET MAUI eine erste Test-Oberfläche erstellt die lediglich einen Text und Button enthält. Spezifischere Funktionen folgen später. Im folgenden seht ihr einige Screenshots, wie die Entwicklung der App auf Desktop und Mobilgerät aussehen wird.

**// Hier seht ihr die Ansicht am Mobilgerät**
![app_android](https://github.com/user-attachments/assets/49183b6b-5fd3-46da-be97-b4088c5b7cd6)

**// Mit Hilfe von Visual Studio können wir die Mobilansicht am Desktop auch simulieren**
![app_android_desktop](https://github.com/user-attachments/assets/18c86819-1512-4932-997f-972410665f03)

**// So sieht die Ansicht nach einfachem öffnen der App auf dem Desktop über die IDE aus**
![app_desktop](https://github.com/user-attachments/assets/57063a13-609a-4fac-a2a4-3c1475b4fd73)

### **Abschluss Sprint 2**
Wie auch nach dem letzten Sprint haben wir uns diese Woche für ein kurzes Retro zusammengesetzt. Wir sind im Großen und Ganzen zufrieden mit dem letzten Sprint. Die nötigen Entwicklungsumgebungen und auch das Git-Repository wurden bei allen aufgesetzt und wir sind startklar mit der Umsetzung der App anzufangen. Nachdem wir nun auch tatsächlich mit etwas Programmierung begonnen haben, sind wir gespannt was im nächsten Sprint auf uns zukommt und stehen der Zukunft unseres Projekts optimistisch gegenüber.

**Comment by Kartoffeleintopf** [comment]
Klingt wie ein guter Start in den dritten Sprint.

> Euer TripAhead-Team (CN)

**Comment by ldcdorn** [comment]
Schönes erstes Design. Unser Team hätten nicht gedacht, dass das so gut mit C# klappen würde . Wir sind gespannt, wie ihr mit C# für die Mobile App Entwicklung langfristig klarkommen werdet. Sehr schon ist es, dass ihr auch über eure Sprints berichtet habt, obwohl das nicht gefordert war, da bekommt man gleich einen viel besseren Eindruck von eurer development Journey. Im Großen und Ganzen eine gelungene Woche!

Viele Grüße
Project HÆLTH
